src/components/NavBar.vue

```diff
@@ -23,6 +23,8 @@
                     <NavBarItem v-for="item of customContent.left" :key="item.key" :item="item" />
                 </b-navbar-nav>
                 <b-navbar-nav id="global-items" class="ml-auto">
+                    <!-- our new search-metadata link -->
+                    <b-nav-item to="/Search2" exact-active-class="active"> Meta-Search </b-nav-item>
```

Comment:
I think this should help the lint error:
```suggestion
                    <b-nav-item to="/search2" exact-active-class="active"> Meta-Search </b-nav-item>
```